### DISCUSSION TITLE
[javascript] 비동기

### DISCUSSION THREAD

**Discussion by lcqff** [BDD-CLUB/challenge-every-week-posting#29] [1. 개발 관련 글 모음]
https://lcqff.github.io/kakaoteccampus/2023/05/25/ch6.html

**Comment by shkisme** [comment]
저는 자바 스크립트를 1도 모르지만... 글 잘 읽었습니다 ㅎㅎ 정리가 아주 잘 되어있네요!!
그나저나 github.io 블로그 진짜 너무 예쁘네요.. 저는 만들다가 포기했었는데 🤣 역시 프론트 개발자님.. 👍
